metadata/InvoicesAustralia__2__metadata.json

```diff
@@ -17,5 +17,5 @@
   "projectName": "UiPath Document Understanding",
   "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
   "tenantName": "UiPath",
-  "imagePath": "registry.replicated.com/aif-core/australianinvoices:2"
+  "imagePath": "registry.replicated.com/aif-core/invoicesaustralia:2"
```

Comment:
Is this change required ? Images are already pushed

Reply:
Also, 1st version might already be present in v20.7.2 release

Reply:
We can retag and push.
We are changing modelNames to match those in prod for:
IndianInvoices => InvoicesIndia
AustralianInvoices => InvoicesAustralia.

Any concerns with this change? Will we need to delete in db?

Reply:
@mohitseth-nitrr

Reply:
we need to make many changes ... and provide a upgrade path ... seems like too much work here